hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/operation/SnapshotOperation.java

```diff
@@ -60,16 +60,16 @@ public SnapshotOperation(long jobId, long executionId, long snapshotId, String m
         ExecutionContext ctx = service.getJobExecutionService().assertExecutionContext(
                 getCallerAddress(), jobId(), executionId, getClass().getSimpleName()
         );
-        CompletableFuture<SnapshotOperationResult> future = ctx.beginSnapshot(snapshotId, mapName, isTerminal)
+        CompletableFuture<SnapshotOperationResult> future = ctx.beginSnapshotPhase1(snapshotId, mapName, isTerminal)
```

Comment:
Wouldn't make more sense to rename `SnapshotOperation` to `SnapshotPhase1Operation` and `SnapshotCompleteOperation` to `SnapshotPhase2Operation`?